src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -284,6 +285,14 @@ export default {
 			return this.messageParameters?.file
 		},
 
+		isNewPollMessage() {
+			if (this.messageParameters?.object?.type !== 'talk-poll') {
+				return false
+			}
+
+			return !!this.$store.getters.getNewPolls[this.messageParameters.object.id]
+		},
+
```

Comment:
Add a condition to check if it is in call. 
Highlighted polls are those that were not voted yet. If yes, then you should check against vote not against toasts because user can close a toast without voting.

Reply:
Added.
Closing the toast shouldn't dismiss the highlight, only opening the Poll. Then, it's up to participant to vote or not.

Reply:
Still, user can open a Poll and close it without voting, so highlight behaves like a highlighter to non-opened poll and not to non-voted poll.

Reply:
Yes, that's intended from my side. We shouldn't oblige user to vote, if he doesn't want to, only to open it and check agenda